### PR TITLE
Add examples to readme

### PR DESCRIPTION
**Pull request by assertchris**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,12 +5,57 @@
...
+
+## Registering the Session Middleware
+
+```php
+$redis = new Amp\Redis\Client("tcp://localhost:6379");
+$mutex = new Amp\Redis\Mutex("tcp://localhost:6379");
+
+$router->use(Aerys\session([
+    "driver" => new Aerys\Session\Redis($redis, $mutex)
+]));
```
Comment (kelunik) [inline README.md]: This should be `$router->use(Aerys\session(new Aerys\Session\Redis($redis, $mutex));` in `v0.2.x`.

File: README.md [hunk 2]
```diff
@@ -5,12 +5,57 @@
...
+## Using the Session
+
+```php
+public function respondToRequest(Request $request, Response $response) {
+    $session = yield (new Aerys\Session($request))->read();
+
+    if (!$session->has("user")) {
+        $response
+            ->setStatus(401)
+            ->end("Unauthorised");
```
Comment (kelunik) [inline README.md]: Missing a `return;` here.

File: README.md [hunk 3]
```diff
@@ -5,12 +5,57 @@
...
+    }
+
+    $user = $session->get("user");
+
+    // ...
+
+    $session->set("token", $token);
```
Comment (kelunik) [inline README.md]: Write access only works after `open()`ing the session. `read()` only opens for reading.

File: README.md (hunk 2 above)
Comment (assertchris) [inline README.md]: Thanks, added below.

File: README.md [hunk 4]
```diff
@@ -5,12 +5,65 @@
...
+}
+```
+
+## Documentation
+
+- [Official Documentation](http://amphp.org/aerys)
+- [Getting Started with Aerys](http://blog.kelunik.com/2015/10/21/getting-started-with-aerys.html)
+- [Getting Started with Aerys WebSockets](http://blog.kelunik.com/2015/10/20/getting-started-with-aerys-websockets.html)
```
Comment (bwoebi) [inline README.md]: not sure how the blog posts are related to the session module?

Comment (assertchris) [inline README.md]: they show the server config around which this session code will be. I can take em out if you like.

**Comment by kelunik** [comment]
Thanks!

**Pull request merged**